python/ray/scripts/scripts.py

```diff
@@ -323,6 +361,10 @@ def start(node_ip_address, redis_address, redis_port, num_redis_shards,
         if no_ui:
             raise Exception("If --head is not passed in, the --no-ui flag is "
                             "not relevant.")
+        if include_java is not None:
```

Comment:
> Can we also check that `java_classpath` is only provided on the head node.

Re:
I think `java_classpath` shouldn't be only set on the head because the classpath on each node may be different. 


> Can we also check that `java_classpath` is only provided if `include_java` is `True`.

Re:
We have actually considered this check, but for the convenience of development, if users don't specify it. we should provide a default `java_classpath`. This way, developers needn't  specify it every time. Otherwise, it's too terrible to specify this every time we debug a cluster.

@robertnishihara Thanks. And what do you think about this? :)

Reply:
@jovany-wang I think what @robertnishihara suggested is `if java_classpath is not None: assert include_java`, not `if include_java: assert java_classpath is not None`

Reply:
@jovany-wang ok, I think providing a default `java_classpath` makes sense. My main suggestion is that if there are any invalid combinations of arguments, make sure we raise an exception here (instead of having something crash later).